homeassistant/components/whirlpool/entity.py

```diff
@@ -64,3 +65,32 @@ def _check_service_request(result: bool) -> None:
                 translation_domain=DOMAIN,
                 translation_key="request_failed",
             )
+
+
+class WhirlpoolOvenEntity(WhirlpoolEntity):
+    """Base class for Whirlpool oven entities."""
+
+    _appliance: Oven
+
+    def __init__(
+        self,
+        appliance: Oven,
+        cavity: OvenCavity,
+        translation_key_base: str | None,
+        unique_id_suffix: str = "",
+    ) -> None:
+        """Initialize the entity."""
+        self.cavity = cavity
+        cavity_suffix = ""
+        if (
+            sum(1 for cavity in OvenCavity if appliance.get_oven_cavity_exists(cavity))
+            > 1
+        ):
```

Comment:
```suggestion
        if appliance.get_oven_cavity_exists(OvenCavity.Upper) and appliance.get_oven_cavity_exists(OvenCavity.Lower):
```
lets simplify this since we only support 2 cavities in the rest of the code anyway